#### Group 7 Members: Tustin Choa, Margaret Inguillo, Ryan Villanueva

In [2]:
import pandas as pd
import matplotlib.pyplot as plt
import numpy as np
import seaborn as sns

In [4]:
stats_df = pd.read_csv("marketing_campaign.csv")

In [7]:
stats_df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2240 entries, 0 to 2239
Data columns (total 1 columns):
 #   Column                                                                                                                                                                                                                                                                                                                                                                    Non-Null Count  Dtype 
---  ------                                                                                                                                                                                                                                                                                                                                                                    --------------  ----- 
 0   ID	Year_Birth	Education	Marital_Status	Income	Kidhome	Teenhome	Dt_Customer	Recency	MntWines	MntFruits	MntMeat

This dataset is about customer personality types. It contains 2240 observations and 29 variables. Those variables are grouped into 4 major categories: People, products, promotion, and place. This was created in order to help companies create products that may be sold to a certain group of people with similar traits. It was likely that the information was gathered through the company's website in the form of cookies, as well as, through membership data. 

Listed below are the variables per category and their interpretation:                           
**People**                                   
ID: Unique number assigned to every customer                               
Year_Birth: Customer's year of birth                                   
Education: Customer's education attainment                             
Marital_Status: Customer's marital status                                
Income: Customer's yearly income                                  
Kidhome: Number of children a customer has                                                       
Teenhome: How many of the children are teenagers                            
Dt_Customer: Customer's date of enrollment at the company                     
Recency: Number of days since customer's last purchase                                
Complain: 1 if a complain was made by the customer in the last 2 years , 0 otherwise                      

**Products**                                                            
MntWines: Expenses on wine in the last 2 years                 
MntFruits: Expenses on fruits in the last 2 years                                    
MntMeatProducts: Expenses on meat in the last 2 years                                   
MntFishProducts: Expenses on fish in the last 2 years                                           
MntSweetProducts: Expenses on sweets the in last 2 years                            
MntGoldProds: Expenses on gold in the last 2 years                       

**Promotion**                                     
NumDealsPurchases: The number of discounted purchases
AcceptedCmp1: 1 if customer bought an offer in the 1st campaign, 0 otherwise                    
AcceptedCmp2: 1 if customer bought an offer in the 2nd campaign, 0 otherwise                     
AcceptedCmp3: 1 if customer bought an offer in the 3rd campaign, 0 otherwise                     
AcceptedCmp4: 1 if customer bought an offer in the 4th campaign, 0 otherwise                              
AcceptedCmp5: 1 if customer bought an offer in the 5th campaign, 0 otherwise                             
Response: 1 if customer bought an offer in the 6th campaign, 0 otherwise                     

**Place**                                          
NumWebPurchases: The number of online purchases                                    
NumCatalogPurchases: The number of purchases through a catalogue                               
NumStorePurchases: The number of purchases made directly in stores                             
NumWebVisitsMonth: The number of visits to company’s web site in the last month                              